src/lib/Code.svelte

```diff
@@ -1,18 +1,38 @@
 <script lang="ts">
 	export let html: string;
 	export let css: string;
+	export let showHTML = false;
+	export let showCSS = false;
 </script>
 
 <section id="code">
 	<h3>CODE</h3>
 
 	<div class="html">
 		<h2>HTML</h2>
-		<pre>{html}</pre>
+		<button on:click={() => (showHTML = !showHTML)}>
+			{#if showHTML}
+				Hide HTML
+			{:else}
+				Show HTML
+			{/if}
+		</button>
+		{#if showHTML}
+			<pre>{html}</pre>
+		{/if}
 	</div>
 	<div>
 		<h2>CSS</h2>
-		<pre>{css}</pre>
+		<button on:click={() => (showCSS = !showCSS)}>
+			{#if showCSS}
+				Hide CSS
+			{:else}
+				Show CSS
+			{/if}
+		</button>
+		{#if showCSS}
+			<pre>{css}</pre>
```

Comment:
Only if you feel like it https://svelte.dev/docs/svelte-transition#slide
```suggestion
			<pre transition:slide>{css}</pre>
```
+ above somewhere `import { slide } from 'svelte/transition';` for easy sleek animation!

Reply:
Ooh nice! I didn't know about this